.github/actions/install-solc/action.yml

```diff
@@ -19,7 +19,7 @@ runs:
       run: |
         mkdir -p ${{ inputs.output-dir }}
         for SOLC_VERSION in ${{ inputs.solc-versions }}; do
-          OUTPUT="${{ inputs.output-dir }}/solc-${SOLC_VERSION}"
+          OUTPUT="${{ inputs.output-dir }}/solc-${SOLC_VERSION%-1.0.1}"
```

Comment:
@antonbaliasnikov maybe you have a better idea 😅

Reply:
@hedgar2017 , I believe we can use `${SOLC_VERSION%%-*}` to filter out everything after `-` that will not require any update when we move to 1.0.2, etc. in the future.

Reply:
I believe we do global replacement, but a good point!